network_acls.tf

```diff
@@ -46,31 +46,34 @@ locals {
       icmp        = null
     }
   ]
+  vpc_inbound_rule = [
+    for zone, addresses in var.address_prefixes : [for index, address in addresses :
```

Comment:
I think the loop should be based on https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/is_vpc_address_prefixes . var.address_prefixes is optional. In the case where it is not specified the address prefixes are determined by the VPC stack, so in any case using the data source to retrieve the actual prefixes from the server is needed.

Reply:
Sure @vburckhardt. I have made the changes to use data source `is_vpc_address_prefix` always for getting address_prefixes.
Pipeline is failing, not sure at the moment what is going wrong here.